spec/lib/openstax/exchange/client/client_configuration_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+describe OpenStax::Exchange::Client, vcr: VCR_OPTS do
+
+  before(:each) do
+    OpenStax::Exchange::Client.configure do |config|
+      config.platform_id     = DEFAULT_CLIENT_PLATFORM_ID
+      config.platform_secret = DEFAULT_CLIENT_PLATFORM_SECRET
+      config.server_base_url = DEFAULT_CLIENT_SERVER_BASE_URL
+      config.server_port     = DEFAULT_CLIENT_SERVER_PORT
+      config.api_version     = DEFAULT_CLIENT_API_VERSION
+    end
+
+    OpenStax::Exchange::Client::FakeClient.configure do |config|
+      config.registered_platforms   = {DEFAULT_CLIENT_PLATFORM_ID => DEFAULT_CLIENT_PLATFORM_SECRET}
+      config.server_base_url        = DEFAULT_CLIENT_SERVER_BASE_URL
+      config.server_port            = DEFAULT_CLIENT_SERVER_PORT
+      config.supported_api_versions = [ DEFAULT_CLIENT_API_VERSION ]
+    end
+  end
+
+  context "internal client instance" do
+    it "#client defaults to a real exchange client" do
+      client = OpenStax::Exchange::Client.send(:client)
+      expect(client.is_real_client?).to be_truthy
```

Comment:
or if you don't want the expectation to be in the helper, the helper can just use an `is_a?` call.